src/Raven.Server/Documents/Indexes/MapReduce/MapReduceIndexingContext.cs

```diff
@@ -20,7 +20,16 @@ public class MapReduceIndexingContext : IDisposable
         public Dictionary<ulong, MapReduceResultsStore> StoreByReduceKeyHash = new Dictionary<ulong, MapReduceResultsStore>(NumericEqualityComparer.BoxedInstanceUInt64);
         public Dictionary<string, long> ProcessedDocEtags = new Dictionary<string, long>();
         public Dictionary<string, long> ProcessedTombstoneEtags = new Dictionary<string, long>();
-        
+
+        public event Action<long> PageModifiedInReduceTree;
+
+        public void OnPageModifiedInReduceTree(long page)
+        {
+            var pageModified = PageModifiedInReduceTree;
+
+            pageModified?.Invoke(page);
```

Comment:
This can be simply `PageModifiedInReduceTree?.Invoke(page)` with the same exact behavior

Reply:
Right. Initially I did it the old way, then applied r# suggestion but didn't remove the local var. I'll change it.